README.md

```diff
@@ -121,55 +232,39 @@ pm2 start --name lisk app.js -- -p [port] -a [address] -c [config-path]
 
 ## Tests
 
-Before running any tests, please ensure Lisk is configured to run on the same testnet that is used by the test-suite.
-
-Replace **config.json** and **genesisBlock.json** with the corresponding files under the **test** directory:
+Run the test suite:
 
 ```
-cp test/config.json test/genesisBlock.json .
+npm run test <test-extent> <test-suite> [test-filename]
 ```
 
-**NOTE:** If the node was started with a different genesis block previous, trauncate the database before running tests.
+**test-extent:** `untagged | unstable | slow | extensive`
 
-```
-dropdb lisk_test
-createdb lisk_test
-```
+**test-suite:** `file | unit | system | transport | api | all`
 
-**NOTE:** The master passphrase for this genesis block is as follows:
+#### Examples:
 
-```
-wagon stock borrow episode laundry kitten salute link globe zero feed marble
-```
+`npm run test file test/unit/blocks.js` - to run a single test-file, specify the path
 
-Launch Lisk (runs on port 4000):
+`npm run test untagged transport` - all untagged (not slow or unstable) transport tests
 
-```
-node app.js
```

Comment:
For functional testing we need to apply less strict behaviour regarding local nodes.
Typically, any connection from a local node will be rejected, but for testing purposes that rule can be ignored.
To disable localhost checking we have to tell the main process that we are working on the test environment by setting the `NODE_ENV` variable.
```
NODE_ENV="test" node app.js
```